src/main/java/uk/gov/hmcts/reform/divorce/orchestration/controller/advice/GlobalExceptionHandler.java

```diff
@@ -59,6 +61,20 @@
             );
     }
 
+    @ExceptionHandler(CaseNotFoundException.class)
+    ResponseEntity<Object> handleCaseNotFoundException(InvalidDataException exception) {
```

Comment:
It would be quite nice to have a handler for the BaseException then the individual handlers would not be needed

Reply:
Yes as discussed on slack will do it separately